src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.Netcore.csproj

```diff
@@ -25,7 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.3.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.5.0-preview" />
```

Comment:
.NET SDK is merged and (with a bit of magic) finally published https://www.nuget.org/packages/Microsoft.Azure.Management.Network/19.4.0-preview 

So now all contributors must change SDK version from local version (localFeed folder) to 19.4.0 and remove nuget packages from localFeed folder.